ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -238,14 +237,24 @@ svi_profiles:
             ip_address_virtual: < IPv4_address/Mask >
             ip_address_virtual_secondaries:
               - < IPv4_address/Mask >
-              - < IPv4_address/Mask >
+
+            # ipv6 address virtual to configure VXLAN Anycast IP address
+            # Optional
+            ipv6_address_virtual: < IPv6_address/Mask >
+            ipv6_address_virtual_secondaries:
+              - < IPv6_address/Mask >
 
             # ip virtual-router address
             # note, also requires an IP address to be configured on the SVI where it is applied.
             # Optional
             ip_virtual_router_addresses:
               - < IPv4_address/Mask | IPv4_address >
-              - < IPv4_address/Mask | IPv4_address >
+
+            # ipv6 virtual-router address
+            # note, also requires an IPv6 address to be configured on the SVI where it is applied.
```

Comment:
Is there a check that verifies that the ipv6 interface address has been specified if we try to use a VARPv6 address?